src/components/Form.js

```diff
@@ -1,41 +1,17 @@
-import React, { useState } from 'react';
-import { useDispatch } from 'react-redux';
-import { v4 as uuidv4 } from 'uuid';
-import { addBook } from '../redux/books/books';
+import PropTypes from 'prop-types';
+import Button from './Button';
 
-const Form = () => {
-  const titleInput = document.getElementById('title');
-  const categoryInput = document.getElementById('category');
-  const dispatch = useDispatch();
-  const [bookState, setBookState] = useState({ id: uuidv4() });
+const Form = ({ handleSubmit }) => (
+  <form>
+    <h3>Add new book</h3>
+    <input name="title" type="text" placeholder="Book title" id="title" required />
+    <input name="author" type="text" placeholder="Author" id="author" required />
+    <Button handleSubmit={handleSubmit} buttonName="Add book" />
+  </form>
+);
```

Comment:
It will be nice to have this component completely handle the functionality of adding a book. Please use this screenshot as a guide to track the unmet requirements.
![form](https://user-images.githubusercontent.com/65476715/135094820-dfc49e0b-2f7f-4daa-b669-94a5f4568161.png)